iceberg/iceberg-handler/src/main/java/org/apache/iceberg/mr/hive/HiveIcebergOutputCommitter.java

```diff
@@ -513,7 +512,7 @@ private void commitTable(FileIO io, ExecutorService executor, OutputTable output
             .map(x -> x.getJobConf().get(IcebergCompactionService.PARTITION_PATH))
             .orElse(null);
 
-        commitCompaction(table, startTime, filesForCommit, rewritePolicy, partitionSpecId, partitionPath);
+        commitCompaction(table, startTime, filesForCommit, partitionSpecId, partitionPath);
```

Comment:
do we still need`rewritePolicy`?

Reply:
Yes, it it's value is still used to control execution in several other places.